### PR TITLE
Upgrade Firefox version

### LINKED ISSUE #13: Travis tests only Chrome
Travis tests only Chrome, is it intentional?

capture d ecran 2016-08-03 a 22 15 13

### PR DESCRIPTION
**Pull request by masonlouchart**
This commit upgrades the version of Firefox used for the tests.

close #13

### CODE REVIEW

**Comment by masonlouchart** [comment]
Nope. Firefox is still not installed...

capture d ecran 2016-08-03 a 22 42 00

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ sudo: required
 dist: trusty
 node_js: stable
 addons:
-  firefox: 46
+  firefox: "48.0"
```
Comment (masonlouchart) [inline .travis.yml]: We can also install the [latest](https://docs.travis-ci.com/user/firefox/).

**Comment by masonlouchart** [comment]
OK. I think it's related to Selenium. I've got the same issue trying to use Firefox 48 locally.

**Comment by saeidzebardast** [comment]
There is a problem in Selenium 9c39b43acb39b6f4398f0102e9aa6c8e3d7b8528.

**Comment by masonlouchart** [comment]
But I don't understand why Travis can't use Firefox. I'm using the version 47.0.1 since its release and everything works.

Sorry for having proposed a PR but I wanted to use Travis (not merge before make the tests pass).

_The version 47.0.0 is not compatible with the last version of web-components-tester._

**Comment by saeidzebardast** [comment]
The Travis job installed the firefox. But as you said, Firefox 47.0.0 is not compatible with the last version of web-components-tester.

**Pull request closed without merging**